### PR TITLE
Add rehanpunjwani to the list

### LINKED ISSUE #730: Add rehanpunjwani
screencapture-github-rehanpunjwani-2021-09-25-18_03_01

### PR DESCRIPTION
**Pull request by rehanpunjwani**
Resolve #730

### CODE REVIEW

**Review by deephunt3r — APPROVED** [review]
LGTM 🚀 Thanks for your contribution 🎉

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -1348,5 +1348,11 @@
     "githubUsername": "shreyaansjain06",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/134031514-b0bbff29-3974-4f05-a2c6-a5c163e90216.png",
     "issueNumber": 726
+  },
+  {
+    "name": "Rehan Punjwani",
+    "githubUsername": "rehanpunjwani",
+    "imageUrl": "https://user-images.githubusercontent.com/49568477/134772561-9e56321a-e6fa-4e85-89a7-52adcb447ffd.png",
```
Comment (Panquesito7) [inline data.json:1355]: I would recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134791010-7e18ffeb-f5f1-4b0f-ac41-4c4bc9006485.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the [**merge conflicts**](https://github.com/EddieHubCommunity/awesome-github-profiles/pull/731/conflicts).

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**